packages/model-viewer/src/features/controls.ts

```diff
@@ -363,6 +365,15 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
       return this[$controls].getFieldOfView();
     }
 
+    // Provided so user code does not have to parse these from attributes.
+    getMinimumFieldOfView(): number|undefined {
+      return this[$controls].options.minimumFieldOfView;
+    }
+
+    getMaximumFieldOfView(): number|undefined {
+      return this[$controls].options.maximumFieldOfView;
```

Comment:
Should I rewrite these as ```return this...maxFov ?? DEFAULT_OPTIONS.maxFov``` then?

Reply:
No, it already returns that if nothing was defined; you can add a test to verify. I'm just saying the type is lying; it will always return a number.